test/domains/compilerErrors/notsupportedBulkAdd-ok.chpl

```diff
@@ -0,0 +1,14 @@
+use notsupportedCommon;
+
+var inds = [1, 2, 3];
+
+//writeln(dfltRectangularBase.bulkAdd(inds));
+//writeln(dfltAssociativeBase.bulkAdd(inds));
+//writeln(DistributedBase.bulkAdd(inds));
```

Comment:
Do we have tests that capture these error messages? I only saw ones for `assoc` and `DR`.

Reply:
As we discussed offline, I am testing the key cases, such as assoc and DR, and skipping the others to reduce testing time.